### ISSUE TITLE
Nested Wrapping is Broken!

### ISSUE THREAD

**Issue by TheLostLambda** [matthijsgroen/ebnf2railroad#55]
Hello! Thanks for the wonderful tool!

I've been using it to work on a new grammar of mine, but I experience some issues when wrapping is turned on! The wrapping is nice and I'd like to use it, but it looks like (when things are wrapped several times / inside of another wrap) it's broken!

Here is my EBNF and a screenshot of the issue:
```
(*
  The PG Structural Language
  ==========================
*)

Multimer = Monomer , {
  ( "~" (* Glycosidic Bond *)
  | "=" (* Peptide Crosslink*)
  ) , Monomer } ;

Monomer = Glycan , [ "-" , Peptide ] ;

Glycan = Monosaccharide , [ Modifications ] ,
  { Monosaccharide , [ Modifications ] } ;

Peptide = AminoAcid , [ Modifications ] ,
  [ LateralChain ] , { AminoAcid , [ Modifications ] ,
  [ LateralChain ] } ;

LateralChain = "[" , Peptide , "]" ;

Monosaccharide = lowercase ;

AminoAcid = uppercase ;

Modifications = "(" , ( "+" | "-" ) , Moiety ,
  { "," , { space } , ( "+" | "-" ) , Moiety } , ")" ;

Moiety = letter , { letter | digit | "_" } ;

(*
  Basic components
  ----------------
  These are low level components, the small building blocks.
*)

space = " " | "\t" ;

letter = uppercase | lowercase ;

uppercase
  = "A" | "B" | "C" | "D" | "E" | "F" | "G"
  | "H" | "I" | "J" | "K" | "L" | "M" | "N"
  | "O" | "P" | "Q" | "R" | "S" | "T" | "U"
  | "V" | "W" | "X" | "Y" | "Z"
  ;

lowercase
  = "a" | "b" | "c" | "d" | "e" | "f" | "g"
  | "h" | "i" | "j" | "k" | "l" | "m" | "n"
  | "o" | "p" | "q" | "r" | "s" | "t" | "u"
  | "v" | "w" | "x" | "y" | "z"
  ;

digit
  = "0" | "1" | "2" | "3" | "4" | "5" | "6"
  | "7" | "8" | "9"
  ;
```

![image](https://user-images.githubusercontent.com/6251883/219866445-932f4078-a7ad-4192-9c5e-9cc3060dd481.png)

Note the mess and broken vertical spacing at the bottom there!

**Comment by TheLostLambda** [comment]
Insane turnaround time! Thanks so much and keep up the stellar work!

I've installed directly from the Github repo, but it would obviously be nice to have the bug-fix on NPM as well (though I'm sure you're already on that!)

Thanks again!

**Comment by matthijsgroen** [comment]
Its published as 1.13.1: https://www.npmjs.com/package/ebnf2railroad
